cl/search/templates/includes/alert_modal.html

```diff
@@ -66,7 +66,7 @@ <h3>
               <p>Select another option or donate to create this alert.</p>
 
               <p class="bottom">
-                <a href="{% url "donate" %}?referrer=rt-alerts"
+                <a href="https://donate.free.law/forms/membership"
                    class="btn btn-danger"
                    target="_blank">Donate Now</a>
```

Comment:
Let's tweak the button with:

```suggestion
                   target="_blank">Join Free.law</a>
```

And the paragraph above should be tweaked to say:

```
<a href="xxx">Real Time alerts</a> are only supported for members of <a href="https://free.law/">Free Law Project</a>.
```

And finally, this line:

>               <p>Select another option or donate to create this alert.</p>

Can be:

>               <p>Please select another rate or become a member to create this alert.</p>